### PR TITLE
renamed remaining control variables

### PR DESCRIPTION
**Pull request by hyder**
Signed-off-by: Ali Mukadam <ali.mukadam@oracle.com>

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -148,25 +149,25 @@ variable "upgrade_operator" {
 
 # operator notification
 
-variable "notification_enabled" {
+variable "operator_notification" {
```
Comment (karthicgit) [inline variables.tf:152]: the variable name has to be operator_notification_enabled

File: README.adoc [hunk 2]
```diff
@@ -38,12 +38,14 @@ The {uri-repo}[Terraform OCI Operator] for {uri-oci}[Oracle Cloud Infrastructure
...
 * A private subnet
-* A compute instance with optional instance_principal access
+* An NSG that allows access only through a bastion host
+* A compute instance:
+** accessible through the OCI Bastion Service
+** with optional instance_principal access
 * An optional notification via email
 
-This module is primarily meant to be reusable and the operator instance is used for:
+This module is meant to be reusabled and to be embedded in your terraform modules. The operator instance is used for:
```
Comment (karthicgit) [inline README.adoc:48]: This module is meant to be reusable

File: README.md [hunk 3]
```diff
@@ -1,41 +1,43 @@
...
 * A private subnet
-* A compute instance with optional instance_principal access
+* An NSG that allows access only through a bastion host
+* A compute instance:
+  - accessible through the OCI Bastion Service
+  - with optional instance_principal access
 * An optional notification via email
 
-This module is primarily meant to be reusable and the operator instance is used for:
+This module is meant to be reusabled and to be embedded in your terraform modules. The operator instance can be used for:
```
Comment (karthicgit) [inline README.md:40]: This module is meant to be reusable

**Review by karthicgit — COMMENTED** [review]
please check the review comments.

File: variables.tf (hunk 1 above)
Comment (hyder) [reply variables.tf:152]: no we are renaming, remember?

File: README.adoc (hunk 2 above)
Comment (hyder) [reply README.adoc:48]: OK

File: variables.tf (hunk 1 above)
Comment (karthicgit) [reply variables.tf:152]: in the doc and changelog its mentioned its getting changed to operator_notification_enabled.

**Review by karthicgit — APPROVED** [review]

**Pull request merged**